core/templates/services/literally-canvas-helper.service.spec.ts

```diff
@@ -0,0 +1,285 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for LiterallyCanvasHelperService.
+ */
+
+describe('LiterallyCanvasHelperService', function() {
+  var LiterallyCanvasHelperService = null;
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.inject(function($injector) {
+    LiterallyCanvasHelperService = $injector.get(
+      'LiterallyCanvasHelperService');
+  }));
+
+  it('should convert a rectangle shapeobject to svg tag', function() {
+    var rectShape = {
+      x: 142.5,
+      y: 96.5,
+      width: 12,
+      height: 29,
+      strokeWidth: 1,
+      strokeColor: 'hsla(0, 0%, 0%, 1)',
+      fillColor: 'hsla(0, 0%, 100%, 1)',
+      id: 'de569866-9c11-b553-f5b7-4194e2380d9f'
+    };
+    var actualSvgTag = '<rect id="rectangle-de569866-9c11-b553-f5b7-4194e23' +
```

Comment:
I feel like I've said this several times already, but: if text spans multiple lines, put an opening paren at the start of the string and break after that paren. Indent subsequent text by 2 lines. Similar to what you did on line 43-44.

Please fix, here and elsewhere, and follow this convention in all future PRs.

Reply:
Done.